lib/src/style/style.dart

```diff
@@ -438,3 +452,7 @@ extension StyleColorList on List {
     }
   }
 }
+
+extension GeoJSONSourceInternal on GeoJsonSource {
+  String? get internalData => '$_data';
```

Comment:
Why this wrapper is needed?

Reply:
It was the only way to get access to the internal `_data` property as the [getter](https://github.com/mapbox/mapbox-maps-flutter/blob/main/lib/src/style/source/geojson_source.dart#L47) returns the value from the style.

Reply:
@pjleonard37 Shouldn't `_data` be accessible from inside the package?

Reply:
Updated this in the latest commit after live discussion. 👍